src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqUserRepository.kt

```diff
@@ -202,6 +202,8 @@ class JooqUserRepository(dsl: DSLContext) : JooqRepository(dsl), UserRepository
             username = newusername
             userGroup = newusername
         }.insert()
+
+        ensureUserHasRole(newusername, ReifiedRole("user", Scope.Global()))
```

Comment:
In order to issue a onetime access token that can be used to authenticate against `password/set`, the user must have the `*/can-login` permission, so we need to give them this on creation.